react/tic-tac-toe/src/app/components/WithLoader/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Spinner from 'react-spinkit';
+
+import styles from './styles.module.scss';
+
+function withLoader(WrappedComponent) {
+  return function Loader({ loader, ...props }) {
+    if (loader) {
```

Comment:
```
return loader ? (
        <div className={styles.container}>
          <Spinner name="double-bounce" fadeIn="none" />
        </div>
      ) : (
      <WrappedComponent {...props} />
     );
```

Reply:
In my opinion I think it is not necessary to use a ternary operator.